app/routes/_gcn.circulars.$circularId.($version)/FrontMatter.tsx

```diff
@@ -40,17 +40,31 @@ export function FrontMatter({
   createdOn,
   submitter,
   submittedHow,
-}: Pick<Circular, 'subject' | 'createdOn' | 'submitter' | 'submittedHow'>) {
+  editedBy,
+  editedOn,
+}: Pick<
+  Circular,
+  | 'subject'
+  | 'createdOn'
+  | 'submitter'
+  | 'submittedHow'
+  | 'editedBy'
+  | 'editedOn'
+>) {
+  const displayDate = editedOn ?? createdOn
   return (
     <>
       <FrontMatterItem label="Subject">{subject}</FrontMatterItem>
       <FrontMatterItem label="Date">
-        {formatDateISO(createdOn)}{' '}
+        {formatDateISO(displayDate)}{' '}
         <small className="text-base-light">
-          (<TimeAgo time={createdOn}></TimeAgo>)
+          (<TimeAgo time={displayDate}></TimeAgo>)
```

Comment:
We should display both the date created and the date revised.

Reply:
@lpsinger do you think it looks better with the dates next to each other:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/5e6574c1-7efa-4879-b980-c155f615ca5a)

or the edit info next to each other:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/94de312b-a995-468c-bf79-668c35744673)

Reply:
Not sure yet: can you show both the original author and the author of the edit, in both styles?

Reply:
The original author is there in both as `From`, what format are you looking for?

Reply:
Oh, I see now. I like the first version.